### ISSUE TITLE
Inspect cannot focus on drop-down editor of Text property

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#3796]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20426.13

* More Info:
This is a not regression issue, it also reproduce on .NET Core 3.1.

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**  [TestPrj.zip](https://github.com/dotnet/winforms/files/5140311/TestPrj.zip)
1. Launch Inspect tool.
2. Create a WinForms core project with PropertyGrid and Button control.
3. Associate the Button with PropertyGrid as a selected object.
4. Build and run the app.
5. Press F4 to open the drop-down editor of **Text** property.
6. Observe the yellow rectangle of inspect.
7. Repeat steps 5-6 several times.

**Actual behavior:**
The yellow rectangle cannot focus on the drop-down editor of Text property.
![Core](https://user-images.githubusercontent.com/62929087/91526420-28c92c80-e936-11ea-8c84-e8d889d0efb8.gif)

**Expected behavior:**
The yellow rectangle should focus on the drop-down editor of Text property.
